packages/internal/src/files.ts

```diff
@@ -78,51 +78,86 @@ export const findPrerenderedHtml = (cwd = getPaths().web.dist) =>
   fg.sync('**/*.html', { cwd, ignore: ['200.html', '404.html'] })
 
 export const isCellFile = (p: string) => {
-  const { dir, name } = path.parse(p)
-  // A Cell must be a directory named module.
-  if (!dir.endsWith(name)) {
+  // If the path isn't on the web side it cannot be a cell
```

Comment:
Did you see an instance where we were passing in files from the api side into this function?

Reply:
Yep - that was what was happening in Curtis' issue. 

See additional tests to check this behaviour

Reply:
So the code that execute this function is the type generation?

`internal/generate/generate.ts -> generateMirrorCells -> findCells`:

The `cwd` is set to the web side? How is this possible?